File: JavaScript/01.JavaScript-Basic-February-2021/04.conditionalStatements-Exercise/02.bonusScore.js
function bonusScore(input) {
    let num = Number(input[0]);

    let bonus = 0;
    if (num <= 100) {
        bonus = 5;
    } else if (num > 1000) {
        bonus = num * 0.10;
    } else {
        bonus = num * 0.20;
    }

    if (num % 2 === 0) {
        bonus += 1;
    } else if (num % 10 === 5) {
        bonus += 2;
    }

    console.log(bonus);
    console.log(bonus + num);
}

bonusScore(["20"]);
bonusScore(["175"]);
bonusScore(["2703"]);
bonusScore(["15875"]);

File: Python/01.Python-Basic-April-2019/05.conditional_statements_exercise/02.bonus_score.py
number = int(input())
bonus = 0

if number <= 100:
    bonus = 5
elif number > 1000:
    bonus = 0.1 * number
else:
    bonus = 0.2 * number

if number % 2 == 0:
    bonus = bonus + 1
elif number % 10 == 5:
    bonus = bonus + 2

print(bonus)
print(bonus + number)